tests/server/tso/common_test.go

```diff
@@ -28,7 +28,7 @@ import (
 )
 
 const (
-	tsoRequestConcurrencyNumber = 5
+	tsoRequestConcurrencyNumber = 3
```

Comment:
why change it?

Reply:
3 clients per keyspace is enough to test multiple clients running concurrently. Previously we creates 5 clients for single group in total. Now we have 5 keyspaces, so creating 3 clients per keyspace and 15 clients should be good enough. 5 * 5 = 25 clients just creates too many logs.